.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "consensus-specs"]
 	path = eth-specs
-	url = git@github.com:ethereum/consensus-specs.git
+	url = https://github.com/ethereum/consensus-specs.git
```

Comment:
If we use the SSH URL instead of `https`, it's failed to run the `git submodule update ...` in some environment. For example, the Github Action of the `nomos-simulations` runs the `pip install ...`, which clones this repo and update submodules. In that environment, the error `Permission denied` occurs. There might be some solutions but I think it's simpler to use the HTTPS URL here.